spec/lib/video_info/providers/vimeo_spec.rb

```diff
@@ -20,6 +20,11 @@
       it { is_expected.to be_truthy }
     end
 
+    context "with Vimeo Review url" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.